src/gc/gccommon.cpp

```diff
@@ -15,10 +15,10 @@
 #include "gc.h"
 
 #ifdef FEATURE_SVR_GC
-SVAL_IMPL_INIT(uint32_t,GCHeap,gcHeapType,GCHeap::GC_HEAP_INVALID);
+SVAL_IMPL_INIT(uint32_t,IGCHeap,gcHeapType,IGCHeap::GC_HEAP_INVALID);
 #endif // FEATURE_SVR_GC
 
-GPTR_IMPL(GCHeap,g_pGCHeap);
+IGCHeapInternal* g_theGcHeap;
```

Comment:
There are two global variables - g_pGCHeap itself got moved to `gcheaputilities.cpp`, where it has the same name and is declared with `GPTR_IMPL`, is there something additional I need to do other than that to make it debuggable?